pkg/ddl/resource_group.go

```diff
@@ -221,6 +221,13 @@ func SetDirectResourceGroupSettings(groupInfo *model.ResourceGroupInfo, opt *ast
 		if len(opt.RunawayOptionList) == 0 {
 			resourceGroupSettings.Runaway = nil
 		}
+		if resourceGroupSettings.Runaway == nil {
+			resourceGroupSettings.Runaway = &model.ResourceGroupRunawaySettings{}
+		} else { // reset the runaway rule settings
+			resourceGroupSettings.Runaway.ExecElapsedTimeMs = 0
+			resourceGroupSettings.Runaway.ProcessedKeys = 0
+			resourceGroupSettings.Runaway.RequestUnit = 0
+		}
```

Comment:
```suggestion
		resourceGroupSettings.Runaway = &model.ResourceGroupRunawaySettings{}
```
Based on the current logic, runaway settings show be set as a whole. Are there any fields need to be inherited?